packages/neutron-sdk/src/errors/error.rs

```diff
@@ -44,6 +47,9 @@ pub enum NeutronError {
 
     #[error("Too many transaction filters, max allowed: {max:?}")]
     TooManyTransactionFilters { max: usize },
+
+    #[error("Can't deconstruct account denom balance key")]
+    AccountDenomBalanceKeyDeconstructionError(String),
```

Comment:
you don't use the String parameter for the error in error formatting

Reply:
Don't get your point, could you elaborate, pls?